sw/device/lib/dif/dif_keymgr.c

```diff
@@ -312,6 +312,25 @@ dif_result_t dif_keymgr_advance_state(const dif_keymgr_t *keymgr,
   return kDifOk;
 }
 
+dif_result_t dif_keymgr_advance_state_raw(const dif_keymgr_t *keymgr) {
+  if (keymgr == NULL) {
+    return kDifBadArg;
+  }
+
+  if (!is_ready(keymgr)) {
+    return kDifLocked;
```

Comment:
maybe this should be `kDifUnavailable`? unless the device is actually locked then nvrmind

Reply:
Probably, but I'll keep this to be consistent with the other "advance"  function if you don't mind. IIRC keymgr config regs are locked when it's busy but I guess this is not what `kDifLocked` means :)